lib/BillTechLinkApiService.php

```diff
@@ -150,6 +149,31 @@ public static function handleBadResponse($response, $path)
 		throw new Exception($message);
 	}
 
+	/**
+	 * @param $customerId
+	 * @param $divisionBankAccount
+	 * @return string
+	 */
+	private static function getBankAccount($customerId, $divisionBankAccount)
+	{
+		global $DB;
+
+		$alternativeBankAccounts = $DB->GetAll(
+			'SELECT contact FROM customercontacts WHERE customerid = ? AND  (type & ?) = ?',
+			array(
+				$customerId,
+				(CONTACT_BANKACCOUNT | CONTACT_INVOICES | CONTACT_DISABLED),
```

Comment:
Co to za konstrukcja?

Reply:
> (type & ?) = ?

i to
> (CONTACT_BANKACCOUNT | CONTACT_INVOICES | CONTACT_DISABLED),
> (CONTACT_BANKACCOUNT | CONTACT_INVOICES)

Reply:
To jest konstrukcja lmsowa, wzięta z pliku ULMS.class.php 71-81 linijki. Sprawdziłem sobie po prostu jak front pobiera te konta bankowe do wypisywania i zrobiłem tutaj tak samo.

Reply:
To coś mi się nie zgadza. Jak mamy:
`(type & ?)` to rzeczywiście oczekujemy `(CONTACT_BANKACCOUNT | CONTACT_INVOICES | CONTACT_DISABLED),`
a jak
` = ?` to chyba nie powinno być `(CONTACT_BANKACCOUNT | CONTACT_INVOICES)`?